docs/semgrep-appsec-platform/gitlab-mr-comments.md

```diff
@@ -107,12 +107,11 @@ For **other CI providers**:
 For more configuration options, see [GitLab CI Sample](/semgrep-ci/sample-ci-configs#gitlab-cicd).
 </details>
 
-:::info MR comments with multiple GitLab groups
-If you're using Semgrep with multiple GitLab groups, ensure that you've completed one of the following steps to ensure that you see MR comments for repositories associated with all of your groups:
+#### MR comments with multiple GitLab groups
+If you're using Semgrep with multiple GitLab groups, ensure that you've completed the following steps to ensure that you see MR comments for repositories associated with all of your groups:
```

Comment:
```suggestion
If you're using [Semgrep with multiple GitLab groups](/deployment/connect-scm#gitlab-cloud), ensure that you've completed the following steps to ensure that you see MR comments for repositories associated with all of your groups:
```

Why **do** we recommend setting up a connection **per** GitLab group?

Reply:
It's the most reliable way to make sure we have access to and necessary metadata for the repos in the group, basically.